src/mouse.h

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright (C) 2015 The University of Notre Dame
+This software is distributed under the GNU General Public License.
+See the file LICENSE for details.
+*/
+
+#ifndef MOUSE_H
+#define MOUSE_H
+
+#include "graphics.h"
+
+#define MOUSE_ACK 0xFA
+// TODO: to support hotplugging, then every so often, need to query if mouse is alive (send 0xEB and recv and ack)
+extern struct graphics_color mouse_fg_color;
+
+int mouse_x;
```

Comment:
This allows people to set the mouse x/y themselves. Is this something we want, or do we want to hide these values behinds getter functions?